CONTRIBUTING.md

```diff
@@ -4,7 +4,7 @@
 Thanks for checking out our contribution guide. We greatly appreciate any work contributed, no matter how small!
 
 ## How to Contribute
-When you are ready to start work on an issue:
+When you are ready to start, work on an issue:
```

Comment:
Intentional as this is a full sentence

Reply:
But what about #1521 sir. I noticed it first and some other guy created a PR

Reply:
Person who created the issue and person who solved the issue need not be the same, `an` was needed, `,` isn't

Reply:
I thought that one to has to create an issue and waited. Then I came to know that for typos one has to create a PR.
I suggested "an" too. You can have look at #1521 .
Yes sir I agree that person who created the issue and person who solved the issue need not be the same, but it was my mistake that I didn't create a PR,